baremetal/metal3machine_manager.go

```diff
@@ -533,8 +533,29 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 			host.Spec.NetworkData = nil
 			bmhUpdated = true
 		}
-		if host.Spec.Online {
+
+		//	turn on/off bmh depending on AutomatedCleaningMode and fastTrack values
+
+		//	Automated Cleaning | Fast Track	| BMH
+		//	disabled			false 		turn off
+		//	disabled			true 		turn off
+		//	metadata			false 		turn off
+		//	metadata 			true 		turn on
+
+		onlineStatus := host.Spec.Online
+
+		//Extract fastTrack value from configMap
+		configMapKey := client.ObjectKey{Name: "capm3-fast-track-cm", Namespace: "capm3-system"}
```

Comment:
parameterize the configmap and namespace parameters

Reply:
done